lib/application.js

```diff
@@ -87,11 +87,16 @@ Application.prototype.isRunning = function () {
 Application.prototype.exists = function () {
   var self = this
   return new Promise(function (resolve, reject) {
-    fs.stat(self.path, function (error, stat) {
-      if (error) return reject(error)
-      if (stat.isFile()) return resolve()
-      reject(Error('Application path specified is not a file: ' + self.path))
-    })
+    if (self.debuggerAddress) {
+      // if debuggerAddress is set, binary (from path) is being ignored by ChromeDriver
+      return resolve();
```

Comment:
Would you mind making this a single line check before the `fs.stat` call that returns early, like:


```js
if (self.debuggerAddress) return resolve()
```

That way they diff doesn't have to indent the `fs.stat` call